### DISCUSSION TITLE
插件【作业平台-构件分发 】里“从远程服务器”求支持填写变量！！！现在只能手动选择节点

### DISCUSSION THREAD

**Discussion by viclab** [Tencent/bk-ci#5171] [Q&A] [answered]
插件【作业平台-构件分发 】里“从远程服务器”求支持填写变量！！！
现在只能手动选择节点，严重影响流程自动化，还有很多插件也有这个问题
![企业微信截图_16316063546881](https://user-images.githubusercontent.com/7496615/133219105-8122e8da-29c3-44ed-a663-af12edca4337.png)

**Comment by jsonwan** [comment, marked as answer]
已排期开发

**Reply by irwinsun** [reply]
<img width="567" alt="image" src="https://user-images.githubusercontent.com/16686129/168553352-a168634d-2faf-4977-9c03-9286be661c71.png">
